geotrek/feedback/models.py

```diff
@@ -44,6 +44,23 @@ def status_default():
     return None
 
 
+class SelectableUserManager(models.Manager):
+
+    def get_queryset(self):
+        return super().get_queryset().filter(userprofile__isnull=False)
+
+
+class SelectableUser(User):
+
+    objects = SelectableUserManager()
+
+    class Meta:
+        proxy = True
+
+    def __str__(self):
+        return f"{self.username} ({self.email})"
```

Comment:
if goal is just allow users with profile only, keep User in FK and defined .filter(userprofile__isnull=False) in form field queryset

Reply:
It's meant to override `__str__` method so that we can see both username and email address in the selector (important since this action will send an email)